src/OrchardCore/OrchardCore/Shell/Removing/ShellSiteFolderRemovingHandler.cs

```diff
@@ -39,6 +43,11 @@ public Task RemovingAsync(ShellRemovingContext context)
         catch (Exception ex) when (ex is DirectoryNotFoundException)
         {
         }
+        catch (Exception ex) when ((ex.HResult & 0x0000FFFF) == 80)
+        {
+            // Sharing violation, may happen if multiple nodes share the same file system
```

Comment:
Shouldn't we at least log this? Same in `ShellWebRootRemovingHandler`.

Reply:
Okay will add it, which log level? Still Error? Or e.g. Warning?

Reply:
I'd use warning since it's recoverable.